### ISSUE TITLE
Collateral

### ISSUE THREAD

**Issue by slepy420** [Berry-Pool/nami-wallet#359]
I am having a problem setting my collateral to do anything within the smart contracts, when I attempt to set my collateral it seems like the nami wallet doesn't see my ledger and keeps saying connect ledger. On the other hand when I try to delegate my Ada Nami wallet sees my ledger and I can confirm my transaction

**Comment by HeptaSean** [comment]
Can confirm that I also see this. #351 sees the same thing for a Trezor.

Perhaps it has something to do with the fact that Ledger cannot be used for smart contract transactions, anyway?

ccvault says: “This is a Ledger account. The Ledger device can't sign smart contract transactions. This feature will be added likely in March.”

The page of the Ledger Cardano app says: “Please note that the current Cardano app available through Ledger Live does not support smart contracts. Our team is currently working on a new version, which you can expect in the coming weeks.”
https://support.ledger.com/hc/en-us/articles/360020095874-Cardano-ADA-?docs=true

Would be good if Nami would also warn the users that contracts with Ledger are not possible at the moment (and therefore collateral setting does not make sense even if it would work).

**Comment by slepy420** [comment]
So is there a way around it? transfer the tokens to a nami address that isn't linked with Ledger maybe?

**Comment by HeptaSean** [comment]
> transfer the tokens to a nami address that isn't linked with Ledger maybe?

Yes, that's what I am doing at the moment: Using the seed phrase wallet that was generated by Nami for the exchange and dApp stuff and only keep the larger amounts of ADA on the Ledger wallet.

**Comment by slepy420** [comment]
Thanks dude!
